mache-chroniclemap/src/main/java/com/excelian/mache/chroniclemap/solr/ArrayIntroSorter.java

```diff
@@ -0,0 +1,58 @@
+package com.excelian.mache.chroniclemap.solr;/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Addendum to the above, if the binary is quite fat or undesirable then we may at least want to copy the solr tests for these classes to cover any future code changes (I think I saw one for the LRU cache at least).